docs/directives/drop.md

```diff
@@ -1,44 +1,44 @@
-# Drop Column
+# Drop
 
-DROP directive is used to drop a column from the record. 
+The `drop` directive is used to drop a column in a record.
 
-## Syntax
 
+## Syntax
 ```
- drop <column>[,<column>]*
+drop <column>[,<column>]*
 ```
 
-```column``` is the name of the column in the record to be droped. If the ```column``` does not exist in the record, then the directive fails.
+The `<column>` is the name of the column in the record to be droped. If the `<column>`
+does not exist in the record, the directive fails.
```

Comment:
technically, if you drop a column that doesn't exist, it just doesn't do anything and continues processing. It doesn't actually give an error message.